### PR TITLE
feat: :sparkles: API call

### PR DESCRIPTION
**Pull request by sjacobs1**
initial API function setup

none

### CODE REVIEW

File: forFakeSake/src/service/getAllProducts.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export function getAllProducts() {
```
Comment (AdeWet) [inline forFakeSake/src/service/getAllProducts.ts]: you can add a return type for this function - you can resolve this comment if needed, I understand this is a first draft :)

**Review by AdeWet — APPROVED** [review]

File: forFakeSake/src/service/getAllProducts.ts
Comment (MarcusNkadimeng) [inline forFakeSake/src/service/getAllProducts.ts:1]: use TanStack query or RTKQ for this when you start working on your product list component.

File: forFakeSake/src/service/getAllProducts.ts (hunk 1 above)
Comment (theadley) [reply forFakeSake/src/service/getAllProducts.ts]: Agreed, this is an implicit any

**Review by theadley — CHANGES_REQUESTED** [review]

File: forFakeSake/src/service/getAllProducts.ts (hunk 1 above)
Comment (sjacobs1) [reply forFakeSake/src/service/getAllProducts.ts]: understood 💪

File: forFakeSake/src/service/getAllProducts.ts
Comment (sjacobs1) [reply forFakeSake/src/service/getAllProducts.ts:1]: will do

**Review by theadley — APPROVED** [review]
Be careful to handle your errors and assumptions properly once you use your query

**Pull request merged**